yoti_python_sdk/sandbox/client.py

```diff
@@ -0,0 +1,184 @@
+            pem_file, "failed in SandboxClient __init__"
+        )
+
+        self.__crypto = Crypto(pem_data)
+
+    def setup_sharing_profile(self, request_token):
+        """
+        Using the supplied YotiTokenRequest, this function will make a request
+        to the defined sandbox environment to create a profile with the supplied values.
+        The returned token can be used against the sandbox environment to retrieve the profile
+        using the standard YotiClient.
+
+        :param YotiTokenRequest request_token:
+        :return: the token for accessing a profile
+        """
+        request_path = self.__endpoint.get_sandbox_path()
+        response = SandboxClient.post(
+            self.__sandbox_url, request_path, self.__crypto, request_token
+        )
+        response_payload = response.json()
+        return YotiTokenResponse(response_payload["token"])
+
+    @staticmethod
+    def builder():
+        """
+        Creates an instance of the sandbox client builder
+
+        :return: instance of SandboxClientBuilder
+        """
+        return SandboxClientBuilder()
+
+    @staticmethod
+    def post(host, path, key, content):
+        payload = json.dumps(content, cls=SandboxEncoder)
+        print(payload)
+        payload_bytes = payload.encode()
+        headers = SandboxClient.__get_request_headers(path, "POST", payload_bytes, key)
+        return requests.post(host + path, payload_bytes, headers=headers, verify=False)
+
+    @staticmethod
+    def __get_request_headers(path, http_method, content, crypto):
+        request = SandboxClient.__create_request(http_method, path, content)
+        sdk_version = yoti_python_sdk.__version__
+
+        return {
+            X_YOTI_AUTH_KEY: crypto.get_public_key(),
+            X_YOTI_AUTH_DIGEST: crypto.sign(request),
+            X_YOTI_SDK: SDK_IDENTIFIER,
+            X_YOTI_SDK_VERSION: "{0}-{1}".format(SDK_IDENTIFIER, sdk_version),
+            "Content-Type": JSON_CONTENT_TYPE,
+            "Accept": JSON_CONTENT_TYPE,
+        }
+
+    @staticmethod
+    def __read_pem_file(key_file_path, error_source):
+        try:
+            key_file_path = expanduser(key_file_path)
+
+            if not isinstance(key_file_path, basestring) or not isfile(key_file_path):
+                raise IOError("File not found: {0}".format(key_file_path))
+            with open(key_file_path, "rb") as pem_file:
+                return pem_file.read().strip()
+        except (AttributeError, IOError, TypeError, OSError) as exc:
+            error = 'Could not read private key file: "{0}", passed as: {1} '.format(
+                key_file_path, error_source
+            )
+            exception = "{0}: {1}".format(type(exc).__name__, exc)
+            raise RuntimeError("{0}: {1}".format(error, exception))
+
+    @staticmethod
+    def __create_request(http_method, path, content):
+        if http_method not in SandboxClient.HTTP_SUPPORTED_METHODS:
```

Comment:
We could change this to use the methods declared in the SDK couldn't we, to avoid declaring it twice? And we can get rid of the declaration in this class
```suggestion
        if http_method not in yoti_python_sdk.client.HTTP_SUPPORTED_METHODS
```